applications/rovioli/app/rovioli-app.cc

```diff
@@ -65,6 +67,29 @@ int main(int argc, char** argv) {
   CHECK(vi_map::getSelectedImu(sensor_manager))
       << "[ROVIOLI] The sensor calibration does not contain an IMU!";
 
+  if (fabs(FLAGS_rovioli_image_resize_factor - 1.0) > 1e-6) {
+    aslam::NCamera::Ptr ncamera = vi_map::getSelectedNCamera(sensor_manager);
```

Comment:
can we have the same convention here as in maplab_node? with `mapping_ncamera` before if and check if its valid?